python_scripts/functions.py

```diff
@@ -408,10 +410,10 @@ def generate_iam_config(
         yaml.dump(out_iam, file)
 
     if iam_policy_path:
-        if iam_policy_path.endwith(".json"):
-            with open(iam_policy_path, "w") as file:
+        if iam_policy_path.endswith(".json"):
```

Comment:
Should we allow `.yaml` as well to write a iam_config instead of policy? Can always just make this an issue and add later on as a "nice to have".

Reply:
Will add an issue, think we discussed this before. Can't remember what we decided on